src/test/js/shared/idam_helper.js

```diff
@@ -701,6 +703,16 @@ class IdamHelper extends Helper {
             console.log(err);
         });
     }
+    async runAccessibilityTest() {
+        if (!testConfig.TestForAccessibility) {
+            return;
+        }
+        const url = await this.helpers[Helper].grabCurrentUrl();
```

Comment:
Helper should be mentioned explicitly i.e. Puppeteer.

Reply:
This has been updated now. Thank you!